lib/Driver/Driver.cpp

```diff
@@ -223,17 +223,19 @@ static void validateDebugInfoArgs(DiagnosticEngine &diags,
     }
   }
 
-  // Check for any -debug-prefix-map options that aren't of the form
+  // Check for any -*-prefix-map options that aren't of the form
   // 'original=remapped' (either side can be empty, however).
   for (auto A : args.getAllArgValues(options::OPT_debug_prefix_map))
```

Comment:
Could we use just one `args.filtered` with `getValues()` + `getOption().getPrefixedName()`?